### DISCUSSION TITLE
Update for tzdata 2022b

### DISCUSSION THREAD

**Discussion by Raman-Iyer** [js-fns/date-fns#3162] [Q&A]
Hello,
Considering the change here:- https://data.iana.org/time-zones/tzdb/NEWS
when are you planning on updating the tzdata database to 2022b?
or is it already done? I cannot find anything related to the same in any discussion, PRs or issues.

**Comment by fturmel** [comment]
Late reply, but I believe this is handled by the JS runtimes (browsers, node.js, etc). No changes required in `date-fns`, these just work through the native `Date` and `Intl` abstractions.
